packages/core/src/Stores/client-store.js

```diff
@@ -1957,16 +1957,20 @@ export default class ClientStore extends BaseStore {
     }
 
     setResidence(residence) {
-        this.accounts[this.loginid].residence = residence;
+        if (this.accounts[this.loginid]) {
+            this.accounts[this.loginid].residence = residence;
+        }
     }
 
     setCitizen(citizen) {
         this.citizen = citizen;
     }
 
     setEmail(email) {
-        this.accounts[this.loginid].email = email;
-        this.email = email;
+        if (this.accounts[this.loginid]) {
+            this.accounts[this.loginid].email = email;
```

Comment:
same here

Reply:
since we assigning values here, it wont work